### ISSUE TITLE
Flaky Dialogflow tests

### ISSUE THREAD

**Issue by averikitsch** [GoogleCloudPlatform/java-docs-samples#1480]
## In which file did you encounter the issue?

dialogflow/cloud-client/src/test/java/com/example/dialogflow/KnowledgeBaseManagementIT.java

## Describe the issue

[Test Fusion](https://fusion.corp.google.com/runanalysis/test/prod%3Acloud-devrel%2Fjava%2Fjava-docs-samples%2Fjava8%2Fperiodic/prod%3Acloud-devrel%2Fjava%2Fjava-docs-samples%2Fjava8%2Fperiodic/KOKORO/569be8a6-9318-4272-a25b-ad463abef9a4/1561316765710/build%20%2375/Targets?target=github%2Fjava-docs-samples%2Fdialogflow%2Fcloud-client%2Ftarget%2Ffailsafe-reports&drilldownTab=tests)

**Comment by nnegrey** [comment]
Nooooooooooo! haha
Um ok, I've tried to make this test less flaky quite a few times, I've got 1 more idea since we now test in Java 8 and 11. 

I'm going to create a permanent Knowledge Base and Document for the DetectIntentKnowledgeBase Method, since I think Dialogflow has a training delay on new documents. If that doesn't work then I will pipe this back to the product team, but I think it is a timing issue.
